packages/twenty-server/src/engine/metadata-modules/object-metadata/interfaces/object-metadata-map.ts

```diff
@@ -0,0 +1,8 @@
+export type ObjectMetadataStandardIdToIdMap = {
```

Comment:
If you use the util mentioned above, you will get a ObjectMetadataItemWithFieldMaps type and won't need this new type

Reply:
See my comment